skins/oasis/css/core/ads.scss

```diff
@@ -227,10 +319,10 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 
 #BOTTOM_LEADERBOARD {
 	background: $color-black;
-	-webkit-transition: height 0.4s ease;
-	-moz-transition: height 0.4s ease;
-	-o-transition: height 0.4s ease;
-	transition: height 0.4s ease;
+	-webkit-transition: height $uap-video-animation-duration ease;
```

Comment:
What do you think about using [the mixin](https://github.com/Wikia/app/blob/6678313298f2ca13ce33c7d205c51c017ea78c6f/skins/shared/mixins/transition.scss) and extending it if necessary?